community/src/main/java/zerobaseproject/community/member/service/MemberService.java

```diff
@@ -0,0 +1,48 @@
+package zerobaseproject.community.member.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+import zerobaseproject.community.global.type.UserRoles;
+import zerobaseproject.community.member.dto.MemberDTO;
+import zerobaseproject.community.member.dto.RegisterDTO;
+import zerobaseproject.community.member.entity.Member;
+import zerobaseproject.community.member.repository.MemberRepository;
+
+@Service
+@RequiredArgsConstructor
+public class MemberService {
+
+    private final MemberRepository memberRepository;
+    private final BCryptPasswordEncoder bCryptPasswordEncoder;
+
+    public MemberDTO signUp(RegisterDTO registerDTO) {
+
+        String email = registerDTO.getEmail();
+
+        if(memberRepository.existsByEmail(email)) {
```

Comment:
```suggestion
        if (memberRepository.existsByEmail(email)) {
```
coding convention 상으로는 if 와 조건문 사이에 중간에 공백이 필요합니다. coding convention 에 대해서도 한번 체화시켜보면 좋을 것 같아요~!
https://naver.github.io/hackday-conventions-java/